app/scripts/directives/overview/listRow.js

```diff
@@ -150,6 +151,8 @@
 
     row.canIDoAny = function() {
```

Comment:
Not new, I'm just adding to the existing behavior.  If we want to come up with some alternative we'll have to do it later, its too late to make a significant change here.

And no pushing this up to DeploymentService.canIDoAny would not be right, this is specific to the actions that we choose to show on the overview.